paasta_tools/cli/cmds/logs.py

```diff
@@ -786,6 +786,9 @@ def filter_and_aggregate_scribe_logs(
         with scribe_reader_ctx as scribe_reader:
             try:
                 for line in scribe_reader:
+                    # temporary until all log lines are strings not byte strings
+                    if isinstance(line, bytes):
+                        line = line.decode()
```

Comment:
`decode('utf8')` ?

Reply:
It's 'utf-8' by default, but I guess an explicit arg is better. I'll make the change.